<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>501</width>
    <height>221</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>481</width>
      <height>201</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::South</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="elideMode">
     <enum>Qt::ElideNone</enum>
    </property>
    <property name="documentMode">
     <bool>false</bool>
    </property>
    <property name="tabBarAutoHide">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="tabSearch">
     <attribute name="title">
      <string>üîé Search</string>
     </attribute>
     <widget class="QComboBox" name="typeSearch">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Word</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Definition</string>
       </property>
      </item>
     </widget>
     <widget class="QLineEdit" name="inputSearch">
      <property name="geometry">
       <rect>
        <x>122</x>
        <y>10</y>
        <width>291</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QListWidget" name="resultList">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>101</width>
        <height>131</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonSearch">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>10</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="definitionBox">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>40</y>
        <width>351</width>
        <height>101</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonEdit">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>150</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>‚úçÔ∏è Edit</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonAddFav">
      <property name="geometry">
       <rect>
        <x>339</x>
        <y>150</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>‚≠êAdd Favorite</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabFavorite">
     <attribute name="title">
      <string>‚≠ê Favorite</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tabHistory">
     <attribute name="title">
      <string>üï¢ History</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tabRandom">
     <attribute name="title">
      <string>üîÄ Random</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tabGame">
     <attribute name="title">
      <string>üéÆ Game</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tabResources">
     <attribute name="title">
      <string>üìú Resources</string>
     </attribute>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
